### PR TITLE
ipabackup: Add playbook tests for ipabackup.

### PR DESCRIPTION
**Pull request by rjeffman**
A new test playbook for ipabackup role can be found at:

    tests/role_backup/test_backup.yml

The test is not yet complete, as 'state: restored' is not tested.

### CODE REVIEW

**Comment by rjeffman** [comment]
This PR is partially related to #893, as it uses the proposed path to store roles playbook tests.

**Review by varunmylaraiah — APPROVED** [review]
Thanks for the PR.
Tests are working fine on downstream.

File: tests/backup_role/test_backup.yml [hunk 1]
```diff
@@ -0,0 +1,418 @@
+---
+- name: Test ansible-freeipa backup role
+  hosts: ipaserver
+  become: yes
+  gather_facts: yes
+
+  tasks:
+  - name: Get current directory.
```
Comment (t-woerner) [inline tests/backup_role/test_backup.yml:8]: What is `pwd` for? Later on `"{{ lookup('env', 'PWD') }}"` is used.

**Comment by t-woerner** [comment]
An additional task to test https://github.com/freeipa/ansible-freeipa/pull/904 would be nice.

File: tests/backup_role/test_backup.yml (hunk 1 above)
Comment (rjeffman) [reply tests/backup_role/test_backup.yml:8]: fixed.

**Pull request merged**